suborbital/reactr
# Add redis implementation of CacheCapability

To test:

First, in a side terminal:
`docker run -it -p "6379:6379" redis`
And then:
`REACTR_TEST_REDIS=true make test`

You'll see 'using Redis for Reactr cache tests` in the test output
